### PR TITLE
[Magiclysm] Update Forge of Wonders Dialogue Options

### LINKED ISSUE #73433: [Magiclysm] - Asking Seer to reveal an Orc Village leads to a Goblin Outpost with the mission "Visit the Orcs"
### Describe the bug

As title says. 

Talking to the seer, it sends the player to a Goblin Outpost (Green "G") instead of a place where Orcs would be. I saw no Orcs there, but killed many a Warg and many a Goblins. 

### Attach save file

N/A

### Steps to reproduce

1. Start a World with Magiclysm
2. Visit Forge of Wanders
3. Talk to Seer
4. Ask for Orcs. 
5. Get Goblins instead.
6. Wonder why "orcs = Goblins". 

### Expected behavior

I would expect to get Orcs. 

### Screenshots

_No response_

### Versions and configuration

Version: Linux 5d07beb

mods:

dda
magiclysm
bombastic_perks

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by TheAthenaCabin0**
#### Summary
Mods "Update Diviner Missions and Forge Merchant Dialogue"

#### Purpose of change
None of the Forge of Wonders Merchants had dialogue indicating when their next restock is, and this resolves that (see #73434). In addition, the diviner at the forge of wonders had some references to places that were out of date (especially the Orc Encampment), closes #73433 .

#### Describe the solution
Edited the json using the pre-existing <interval> call for the restock dialogue options, and also edited the json dialogue of the diviner to make all the quest options do what they say they will do. 

#### Describe alternatives you've considered
Leaving the dialogue as is and letting people remain confused as to what the options really mean in game.

#### Testing
No testing was done (I had issues with compiling it completely unrelated to the changes I made)

#### Additional context
None I think.

### CODE REVIEW

**Comment by Standing-Storm** [comment]
<3 to the goblin-related dialogue

File: data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json [hunk 1]
```diff
@@ -115,5 +116,19 @@
...
+    "dynamic_line": {
+      "u_is_wearing": "badge_marshal",
+      "yes": [
+        "Do you think me a fool, to tell an officer of the law my current plans?",
+        "Not even trying to ask me that undercover? By the seventeen seas, officers of the law really ARE made worse these days.",
+        "The end of times is here and that's what you're worried about? You should get your priorities straight."
+      ],
+      "no": "Well you obviously aren't an officer so here's the deal. There's a mighty fine galleon making land in <interval>, I'm certain to have something good by then."
+    },
+    "responses": [ { "text": "May the seas be ever in your favor", "topic": "TALK_FORGE_LORD_REAGENT" } ]
```
Comment (NetSysFire) [inline data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json:132]:
```suggestion
    "responses": [ { "text": "May the seas be ever in your favor.", "topic": "TALK_FORGE_LORD_REAGENT" } ]
```

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json [hunk 2]
```diff
@@ -508,15 +515,15 @@
     ]
   },
   {
-    "id": "TALK_FORGE_LORD_DIVINER_ORC",
+    "id": "TALK_FORGE_LORD_DIVINER_GOBLIN_OUTPOST",
     "type": "talk_topic",
-    "dynamic_line": "Militaristic xenophobes, that bunch.  They are fairly good weapon smiths, and quite strong, so be careful.  Oh, and something about their physiology makes them extra super dangerous when raised as undead.  So you should probably try to not let that happen.  I'll mark one of their camps on your map for one denarius.",
+    "dynamic_line": "Contrary to popular belief, goblins are not actually orcs.  Its the other way around.  Orcs are goblins who have taken one too many unlabeled potions and lived to tell the tale.  Sometimes this awakens powers much like my own.  Well, not exactly, if it did I'd be out of a job faster than a Magus could cast Phase Door.  Luckily for you, you shouldn't have to delve too deep into a forest to see for yourself. I'll mark one of their outposts on your map for one denarius.",
```
Comment (NetSysFire) [inline data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:520]: This feels off compared to the other additions. Maybe a bit too educational?

**Review by NetSysFire — APPROVED** [review]
Overall I like how the characters personalities are being extended meaningfully.

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json (hunk 2 above)
Comment (TheAthenaCabin0) [reply data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:520]: The reason I went with this more educational spiel was trying to have ut be more in line with a few of the other dialogue she has when giving monster related missions, with a little tip on what to expect. However I might have gone a little bit overboard especially in the beginning, but I wanted to get a few of those little flavor regarding Orcs that the author of the giblin/orc overhaul desired into the game

File: data/mods/Magiclysm/npc/TALK_FORGE_MERCHANT.json [hunk 3]
```diff
@@ -92,6 +92,7 @@
       { "text": "Let's trade.", "effect": "start_trade", "topic": "TALK_FORGE_LORD" },
       { "text": "What are you doing here?", "topic": "TALK_FORGE_LORD_DOING" },
       { "text": "I have some stories for you.", "topic": "TALK_FORGE_LORD_BOOKS_DELIVER" },
+      [ "text": "Anything interesting being worked on?", "topic": "TALK_FORGE_LORD_ASK_INTERVAL" },
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_MERCHANT.json:95]:
```suggestion
      { "text": "Anything interesting being worked on?", "topic": "TALK_FORGE_LORD_ASK_INTERVAL" },
```

**Comment by Maleclypse** [comment]
Ok that error was almost invisible. a bracket instead of a curlyque.

File: data/mods/Magiclysm/npc/TALK_FORGE_ARDELIA.json [hunk 4]
```diff
@@ -68,17 +68,24 @@
...
     "id": "TALK_FORGE_LIBRARIAN_DOING",
     "type": "talk_topic",
-    "dynamic_line": "I organize the books, I sell extra books, once when we still lent books I would hunt down those with late books to return.",
+    "dynamic_line": "I organize the books, I sell extra books, back when we still lent books I would hunt down those with late books to return.  I have done this for aeons past and I will continue to do this for aeons to come.",
     "responses": [ { "text": "Let's talk business then.", "topic": "TALK_FORGE_LORD_LIBRARIAN" } ]
+  },
+  {
+    "id": "TALK_FORGE_LIBRARIAN_ASK_INTERVAL",
+    "type": "talk_topic",
+    "dynamic_line": "It shouldn't take Valzain too long to read what he desires from what I have out. Check back in <interval> and I should have some new books set out.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_ARDELIA.json:88]:
```suggestion
    "dynamic_line": "It shouldn't take Valzain too long to read what he desires from what I have out.  Check back in <interval> and I should have some new books set out.",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json [hunk 5]
```diff
@@ -42,7 +42,7 @@
   {
     "id": "TALK_FORGE_LORD_DIVINER",
     "type": "talk_topic",
-    "dynamic_line": "Are you looking for something in particular?",
+    "dynamic_line": "Are you looking for something? I can help with anything but your own stuff. You'll have to ask your mother for help with that.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:45]:
```suggestion
    "dynamic_line": "Are you looking for something?  I can help with anything but your own stuff. You'll have to ask your mother for help with that.",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json [hunk 6]
```diff
@@ -52,7 +52,7 @@
   {
     "id": "TALK_FORGE_DIVINER_DOING",
     "type": "talk_topic",
-    "dynamic_line": "I specialize in scrying.  I can help you find any number of things.",
+    "dynamic_line": "What are any of us really doing here in this dead and dying world? I specialize in scrying, and while I wish I could say I'm in it to help lost souls find their way, the collective pockets of a thousand realities is a very good incentive. I can help you find any number of things, but I take no responsibility for what happens when you find what you're looking for.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:55]:
```suggestion
    "dynamic_line": "What are any of us really doing here in this dead and dying world?  I specialize in scrying, and while I wish I could say I'm in it to help lost souls find their way, the collective pockets of a thousand realities is a very good incentive.  I can help you find any number of things, but I take no responsibility for what happens when you find what you're looking for.",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json [hunk 7]
```diff
@@ -105,16 +103,15 @@
   {
     "id": "TALK_FORGE_LORD_DIVINER_MAN_MADE",
     "type": "talk_topic",
-    "dynamic_line": "Seeking to pick around your civilization's corpse?",
+    "dynamic_line": "Seeking to pick around your civilization's corpse? You're not the first and you won't be the last.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:106]:
```suggestion
    "dynamic_line": "Seeking to pick around your civilization's corpse?  You're not the first and you won't be the last.",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json [hunk 8]
```diff
@@ -490,13 +486,24 @@
   {
     "id": "TALK_FORGE_LORD_DIVINER_GOBLIN",
     "type": "talk_topic",
-    "dynamic_line": "Good choice.  Small, angry creatures obsessed with trash, yet with a tendency to hoard some surprisingly valuable artefacts.  Just watch out for the mind goblin.  Actually, uh, never mind.  I'll mark one of their camps on your map for one denarius.",
+    "dynamic_line": "Good choice.  It seems that shortly before the world started to end, a few of them got it in their heads to just leave civilization alone for a bit and go live in the woods with other goblins. And wargs.  Not a terrible idea if you ask me.  Anyways, theres a few different types of places they gather, are you looking for an encampment or an outpost?",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:489]:
```suggestion
    "dynamic_line": "Good choice.  It seems that shortly before the world started to end, a few of them got it in their heads to just leave civilization alone for a bit and go live in the woods with other goblins.  And wargs.  Not a terrible idea if you ask me.  Anyways, theres a few different types of places they gather, are you looking for an encampment or an outpost?",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json [hunk 9]
```diff
@@ -490,13 +486,24 @@
...
+      { "text": "An encampment.", "topic": "TALK_FORGE_LORD_DIVINER_GOBLIN_ENCAMPMENT" },
+      { "text": "An outpost.", "topic": "TALK_FORGE_LORD_DIVINER_GOBLIN_OUTPOST" },
+      { "text": "On second thought…", "topic": "TALK_NONE" },
+      { "text": "Nevermind.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_FORGE_LORD_DIVINER_GOBLIN_ENCAMPMENT",
+    "type": "talk_topic",
+    "dynamic_line": "You've got a long walk deep into the forest ahead of you.  Oh, and did I mention the wargs?  They've got these wargs that are absolutely adorable. Of course they'll maul you if you try to pet them, but the temptation is always there.  I'll mark one of their camps on your map for one denarius.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:500]:
```suggestion
    "dynamic_line": "You've got a long walk deep into the forest ahead of you.  Oh, and did I mention the wargs?  They've got these wargs that are absolutely adorable.  Of course they'll maul you if you try to pet them, but the temptation is always there.  I'll mark one of their camps on your map for one denarius.",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json [hunk 10]
```diff
@@ -42,7 +42,7 @@
   {
     "id": "TALK_FORGE_LORD_DIVINER",
     "type": "talk_topic",
-    "dynamic_line": "Are you looking for something in particular?",
+    "dynamic_line": "Are you looking for something?  I can help with anything but your own stuff. You'll have to ask your mother for help with that.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:45]:
```suggestion
    "dynamic_line": "Are you looking for something?  I can help with anything but your own stuff.  You'll have to ask your mother for help with that.",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json (hunk 2 above)
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_DIVINER.json:520]:
```suggestion
    "dynamic_line": "Contrary to popular belief, goblins are not actually orcs.  Its the other way around.  Orcs are goblins who have taken one too many unlabeled potions and lived to tell the tale.  Sometimes this awakens powers much like my own.  Well, not exactly, if it did I'd be out of a job faster than a Magus could cast Phase Door.  Luckily for you, you shouldn't have to delve too deep into a forest to see for yourself.  I'll mark one of their outposts on your map for one denarius.",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json [hunk 11]
```diff
@@ -115,5 +116,19 @@
...
     "responses": [ { "text": "Well, shiver me timbers then.", "topic": "TALK_FORGE_LORD_REAGENT" } ]
+  },
+  {
+    "id": "TALK_FORGE_REAGENT_ASK_INTERVAL",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_is_wearing": "badge_marshal",
+      "yes": [
+        "Do you think me a fool, to tell an officer of the law my current plans?",
+        "Not even trying to ask me that undercover? By the seventeen seas, officers of the law really ARE made worse these days.",
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json:127]:
```suggestion
        "Not even trying to ask me that undercover?  By the seventeen seas, officers of the law really ARE made worse these days.",
```

File: data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json [hunk 12]
```diff
@@ -115,5 +116,19 @@
...
+        "The end of times is here and that's what you're worried about? You should get your priorities straight."
+      ],
+      "no": "Well you obviously aren't an officer so here's the deal. There's a mighty fine galleon making land in <interval>, I'm certain to have something good by then."
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json:130]:
```suggestion
      "no": "Well you obviously aren't an officer so here's the deal.  There's a mighty fine galleon making land in <interval>, I'm certain to have something good by then."
```

File: data/mods/Magiclysm/npc/TALK_FORGE_MERCHANT.json [hunk 13]
```diff
@@ -135,7 +136,7 @@
       "u_has_var": "general_meeting_u_met_FORGE_LORD",
       "value": "yes",
       "no": "Well here and now I am the law, and the law here is that people purchase my goods, they take them out into the world and they die; or they return to buy more of my goods.  Either way, I write poetry about them.",
-      "yes": "Once I was lost in the throes of love, now I've built an empire of material goods."
+      "yes": "Once I was lost in the throes of love, now I've built an empire of material goods. I've found these two things are equally as powerful, and dangerous."
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_MERCHANT.json:139]:
```suggestion
      "yes": "Once I was lost in the throes of love, now I've built an empire of material goods.  I've found these two things are equally as powerful, and dangerous."
```

File: data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json [hunk 14]
```diff
@@ -115,5 +116,19 @@
...
+  },
+  {
+    "id": "TALK_FORGE_REAGENT_ASK_INTERVAL",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_is_wearing": "badge_marshal",
+      "yes": [
+        "Do you think me a fool, to tell an officer of the law my current plans?",
+        "Not even trying to ask me that undercover?  By the seventeen seas, officers of the law really ARE made worse these days.",
+        "The end of times is here and that's what you're worried about? You should get your priorities straight."
```
Comment (Maleclypse) [inline data/mods/Magiclysm/npc/TALK_FORGE_HELEN_TAVREL.json:128]:
```suggestion
        "The end of times is here and that's what you're worried about?  You should get your priorities straight."
```

**Pull request merged**